src/generators/web.cr

```diff
@@ -162,7 +162,7 @@ class LuckyCli::Generators::Web
         version: ~> 0.1.2
       dotenv:
```

Comment:
@jwoertink Do you think we take this release to move to `lucky_env`?

Reply:
I thought about that. Maybe another PR?